docs/src/man/charge_drift.md

```diff
@@ -261,6 +261,59 @@ parameters = Dict("parameters" =>
 sim.detector = SolidStateDetector(sim.detector, BoggsChargeTrappingModel{T}(parameters))
 ```
 
+### `ConstantLifetimeChargeTrappingModel`
+This constant-lifetime-based charge trapping model assumes electrons and holes to have a constant free lifetime throughout the crystal.
+In this model, the charge cloud loses part of its charge at every point `path[i]` of the charge drift, depending on the lifetime.
+The charge signal is then given by the charge-decreased charge cloud reaching the contacts and the charges trapped on the way.
+
+This idea is presented in [Dai _et al._ (2023)](https://doi.org/10.1016/j.apradiso.2022.110638).
+
+Besides, the model implemented has two sets of parameters for the sensitive (bulk) and inactive (dead layer/surface layer) volume respectively.
+The inactive layer effect will only be considered if the corresponding `virtual_drift_volume` is defined in the configuration file.
+
+The `ConstantLifetimeChargeTrappingModel` can be applied in the configuration file by adding a field `charge_trapping_model` to the `semiconductor` with `model: ConstantLifetime`, `parameters` defining the constant lifetime and `inactive_layer_geometry`(optional) defining the inactive volume:
+```yaml 
+detectors:
+  - semiconductor:
+      material: #...
+      geometry: #...
+      charge_trapping_model:
+        model: ConstantLifetime
+        parameters:
+          τh: 1ms
+          τe: 1ms
+          τh_inactive: 80ns
+          τe_inactive: 80ns
+        inactive_layer_geometry:
+          tube:
+            r:
+              from: 9.0
+              to: 10.0
+            h: 10.0
+            origin:
+              z: 5.0
+```
+
+The inactive layer could only be modeled with the help of the configuration file.
+But for the sensitive volume, the parameters can always be applied to an already read-in `SolidStateDetector` using for example:
+```julia
+using SolidStateDetectors, Unitful
+T = Float64
+sim = Simulation{T}(SSD_examples[:TrueCoaxial])
+simulate!(sim)
+
+# only model the sensitive volume
+τh = τe = 1u"ms"
+parameters = Dict("parameters" => Dict("τh" => τh, "τe" => τe))
+
+# model both the sensitive and inactive volumes based on the inactive volume geometry defined in the configuration file
+τh = τe = 1u"ms"
+τh_inactive = τe_inactive = 80u"ns"
+parameters = Dict("parameters" => Dict("τh" => τh, "τe" => τe, "τh_inactive" => τh_inactive, "τe_inactive" => τe_inactive), "inactive_layer_geometry" => sim.detector.semiconductor.charge_trapping_model.inactive_layer_geometry)
+
+sim.detector = SolidStateDetector(sim.detector, ConstantLifetimeChargeTrappingModel{T}(parameters))
```

Comment:
Also here, it looks like you can pass a geometry, even without using a configuration file.